plugins/source/aws/client/data/partition_service_region.json

```diff
@@ -260,6 +260,12 @@
             "us-west-2": {}
           }
         },
+        "api.ecr-public": {
+          "regions": {
+            "us-east-1": {},
+            "us-west-2": {}
+          }
+        },
```

Comment:
Hmm... assuming we got this file from somewhere, this can cause issues the next time we update it. Where do we get it from? Is it provided by AWS? Is ecr-public also not included in the latest version? 

Again assuming that this file was provided to us somehow, if we have to make a manual change, I think we should put it in a separate JSON file (called `paritition_service_region_overrides.json` or something like that) so we can be explicit about this being an addition that we made.

Reply:
(See my later comment lower down - seems like my concerns were addressed)